modules/buttons/GooglePlusButtonByRakesh-116.html

```diff
@@ -0,0 +1,20 @@
+<div>
+  <style>
+      .google-button{
+        background-color:#db4d4d;
+        padding:10px;
+        width:120px;
+        height:45px;
+        border-width: 0;
+        border-radius:2px;
+        color:#ffffff;
+        font-size:15px;
+        font-weight: 100;
+      }
+    </style>
+    <div>
+        <button class="google-button">
+            <img src="Google_Plus_logo_(2015-2019).svg.png"/>Google +
```

Comment:
I can't seem to find this image location? Did you actually need it? 

**Story Time**
In this "Session of Buttons", I saw a guy who designed the entire button without an svg or any img. I was wowed. Just some Text span and nice styling. lol....

Reply:
I have done the changes according to your query..